src/features/features.jl

```diff
@@ -20,9 +20,11 @@ using ..ChemistryFeaturization.AbstractType:
     AbstractAtoms, AbstractFeatureDescriptor, AbstractCodec
 using ..ChemistryFeaturization.Codec: OneHotOneCold
 
+# pretty printing, short version
+Base.show(io::IO, fd::AbstractFeatureDescriptor) = print(io, "$(typeof(fd)) $(fd.name)")
+
 import ..ChemistryFeaturization.encodable_elements
-encodable_elements(fd::AbstractFeatureDescriptor) =
-    throw(MethodError(encodable_elements, fd))
+encodable_elements(fd::AbstractFeatureDescriptor) = fd.encodable_elements
```

Comment:
Yeah fair enough I guess. I did it because at present all of them except for EFD do have such a field, but I suppose the alternative is more idiomatic